### PR TITLE
Tc/payment plan drop

### PR DESCRIPTION
**Pull request by tomchipchase**
Added the docs for payment plans.

I also spent a few minutes googling the correct command to run in order to check this locally, so I added a short readme.

### CODE REVIEW

**Comment by tomchipchase** [comment]
[Add drop for payment plans](https://trello.com/c/GIAYWody/232-add-drop-for-payment-plans)

File: README [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Development
```
Comment (RyanMacG) [inline README:1]: Good spot, didn't realise we didn't have this documented!

File: docs/reference/objects/product/payment_plan.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: PaymentPlan
+parent: Product
+has_children: false
+---
+
+A `payment_plan` has the following attributes:
+
+# payment_plan.number_of_instalments
```
Comment (RyanMacG) [inline docs/reference/objects/product/payment_plan.md:10]: [Suggestion] I think we've alphabetised these elsewhere so might be worth following that convention.

**Review by RyanMacG — APPROVED** [review]
Minor suggestion but otherwise LGTM

**Pull request merged**